lib/gui/components/update-notifier/services/update-notifier.js

```diff
@@ -60,7 +60,9 @@ module.exports = function($http, $q, ModalService, UPDATE_NOTIFIER_SLEEP_TIME, M
           // The error status equals -1 if the request couldn't
           // be made successfully, for example, because of a
           // timeout on an unstable network connection.
-          if (error.status === -1) {
+          const ERROR_CODE_UNSUCCESSFULL_REQUEST = -1;
```

Comment:
> this error code is only relevant in the context of the `etcherLatestVersion` function.

In that case maybe the same module that exports the `etcherLatestVersion` function could also export appropriate constants for the error codes? :)